noxfile.py

```diff
@@ -101,3 +124,40 @@ def docs(session):
         os.path.join("docs", ""),
         os.path.join("docs", "_build", "html", ""),
     )
+
+
+@nox.session(python=DEFAULT_PYTHON_VERSION)
+def docfx(session):
+    """Build the docfx yaml files for this library."""
+
+    session.install("-e", ".[tracing]")
+    # sphinx-docfx-yaml supports up to sphinx version 1.5.5.
+    # https://github.com/docascode/sphinx-docfx-yaml/issues/97
+    session.install(
+        "sphinx==1.5.5", "alabaster", "recommonmark", "sphinx-docfx-yaml"
+    )
```

Comment:
Thanks for adding this! Small tweak to match the current template: https://github.com/googleapis/synthtool/blob/063de45298fbdd88916018ba566c7ecd254b39ae/synthtool/gcp/templates/python_library/noxfile.py.j2#L260

```suggestion
    session.install(
        "sphinx", "alabaster", "recommonmark", "gcp-sphinx-docfx-yaml"
    )
```

Reply:
made the change, thanks :)